Zend/zend_enum.c

```diff
@@ -20,6 +20,7 @@
 #include "zend_API.h"
 #include "zend_compile.h"
 #include "zend_enum_arginfo.h"
+#include "zend_interfaces_arginfo.h"
```

Comment:
@kocsismate I'm importing this to have access to `arginfo_class_Stringable___toString` but this generates a bunch of "defined but not used" warnings. How would you fix those? Split the stubs?

Reply:
Yes, arginfo.h files should only be imported once, so splitting the required parts would make sense.

Reply:
@kocsismate Note: The stringable stubs would still be imported twice, but there wouldn't be any other symbols included twice and not used in this case. Is that ok? The problem is, I can't declare the function in an enum interface since `__toString` is only generated for string enums. And I want to avoid creating another interface.